content/applications/hr/employees.rst

```diff
@@ -0,0 +1,14 @@
+:nosearch:
+:show-content:
+:show-toc:
+
+=========
+Employees
+=========
+
+Odoo **Employees** organizes a company's employee records, contracts, and departments.
```

Comment:
```suggestion
Odoo *Employees* organizes a company's employee records, contracts, and departments.
```
It's my understanding that we are meant to italicize the name of the app - not bold

Reply:
Ahh - that's right! This doc was copied off of other docs, so that's why it was just bold.